platform-enterprise_docs/compute-envs/aws-cloud.md

```diff
@@ -180,14 +181,18 @@ The following permissions enable Seqera to populate values for dropdown fields.
 }
 ```
 
-## Managed Amazon Machine Image (AMI) 
+## Managed Amazon Machine Image (AMI)
 
-The AWS Cloud compute environment uses an AMI maintained by Seqera, and the pipeline launch procedure assumes that some basic tooling is already present in the image itself. If you want to provide your own AMI, it must include at least the following:
+The AWS Cloud compute environment uses a public AMI maintained by Seqera, and the pipeline launch procedure assumes that some basic tooling is already present in the image itself. If you want to provide your own AMI, it must include at least the following:
 
 - Docker engine, configured to run at startup.
 - CloudWatch agent.
 - The ability to shut down with the `shutdown` command. If this is missing, EC2 instances will keep running and accumulate additional costs.
 
+### Release cadence and sofware updates
+
+The AMI is based off the [AmazonLinux2023 image](https://docs.aws.amazon.com/linux/al2023/ug/what-is-amazon-linux.html): system package versions for a specific AmazonLinux2023 version are pinned. We subscribe to the [AWS SNS topic](https://docs.aws.amazon.com/linux/al2023/ug/receive-update-notification.html) to receive AmazonLinux2023 update notification and trigger a Seqera AMI release on top of the latest released image, bringing sytem package updates and security patches.
```

Comment:
```suggestion
The AMI is based on the [Amazon Linux 2023 image](https://docs.aws.amazon.com/linux/al2023/ug/what-is-amazon-linux.html). System package versions are pinned for each specific Amazon Linux 2023 version. Seqera subscribes to the [AWS SNS topic](https://docs.aws.amazon.com/linux/al2023/ug/receive-update-notification.html) to receive Amazon Linux 2023 update notifications. When updates are available, this triggers a new Seqera AMI release built on the latest image, which includes system package updates and security patches.
```
Please check that the meaning is retained.

Reply:
Thanks for the improvement! Looks good to me, I'll apply the suggestion :+1: